openpype/modules/slack/plugins/publish/integrate_slack_api.py

```diff
@@ -377,8 +381,12 @@ def send_message(self, channel, message, publish_files):
             return response.data["ts"], file_ids
         except SlackApiError as e:
             # # You will get a SlackApiError if "ok" is False
-            error_str = self._enrich_error(str(e.response["error"]), channel)
-            self.log.warning("Error happened {}".format(error_str))
+            if e.response.get("errror"):
+                error_str = self._enrich_error(str(e.response["error"]), channel)
+            else:
+                error_str = self._enrich_error(str(e), channel)
+            self.log.warning("Error happened {}".format(error_str),
```

Comment:
Match error message formattign with the ones below.
```suggestion
            self.log.warning("Error happened: {}".format(error_str),
```